x-pack/test/functional/page_objects/rollup_page.js

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from 'expect.js';
+import { map as mapAsync } from 'bluebird';
+
+export function RollUpPageProvider({ getService, getPageObjects }) {
```

Comment:
I think any instances of `RollUp`/`rollUp` prefix (i.e. uppercase `U`) should be changed to `Rollup`/`rollup` for consistency with the rest of code